training/tf/parse.py

```diff
@@ -65,28 +104,50 @@ def __init__(self, chunks, workers=None):
         # Build the all-zeros and all-ones flat planes, used for color-to-move.
         self.flat_planes = [ b'\0' * 361, b'\1' * 361 ]
 
+        # set the down-sampling rate
+        self.sample = sample
+
+        # V2 Format (see 'PackedMove' in Training.h)
```

Comment:
I think this part of the pull request is missing.

Reply:
I removed the mention. If you're interested in those changes, I'll put them into a separate pull request (that code is still a little messier, and not backward compatible yet).